### PR TITLE
9.0 add ccount invoice journal filter

### PR DESCRIPTION
**Pull request by susport**
This module adds possibility of filtering by journal on invoices.

### CODE REVIEW

File: account_invoice_journal_filter/README.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Account Invioce Journal Filter
+==============================
+
+This module extends the functionality of account invoice to filter by to journal.
```
Comment (jjscarafia) [inline account_invoice_journal_filter/README.rst]: remove "to"
Should be "This module extends functionality of account invoice to filter by journal"

File: account_invoice_journal_filter/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Account Invoice Journal Filter",
+    'version': '9.0.1.0.0',
+    'category': 'Accounting',
+    'sequence': 14,
+    'summary': 'Quotations, Sales Orders, Invoicing',
+    'description': """
```
Comment (jjscarafia) [inline account_invoice_journal_filter/__openerp__.py]: Remove description from here, already on README.rst

File: account_invoice_journal_filter/README.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Firstname Lastname <jjs@adhoc.com.ar>
+
```
Comment (houssine78) [inline account_invoice_journal_filter/README.rst:41]: give the firstname and the lastname of the contributor except if those are his real firstname and lastname.

File: account_invoice_journal_filter/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    ],
+    'data': [
+        'invoice_view.xml',
+    ],
+    'demo': [
+    ],
+    'test': [
+    ],
+    'installable': True,
+    'auto_install': False,
```
Comment (houssine78) [inline account_invoice_journal_filter/__openerp__.py:43]: remove the line as False is the default value

File: account_invoice_journal_filter/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    'depends': [
+        'account',
+    ],
+    'data': [
+        'invoice_view.xml',
+    ],
+    'demo': [
+    ],
+    'test': [
+    ],
```
Comment (houssine78) [inline account_invoice_journal_filter/__openerp__.py:41]: remove empty key

File: account_invoice_journal_filter/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Account Invoice Journal Filter",
+    'version': '9.0.1.0.0',
+    'category': 'Accounting',
+    'sequence': 14,
+    'summary': 'Quotations, Sales Orders, Invoicing',
+    'author':  'Odoo Community Association (OCA)',
```
Comment (houssine78) [inline account_invoice_journal_filter/__openerp__.py:27]: AFAIU is only related to Invoicing

File: account_invoice_journal_filter/README.rst [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+Configuration
+=============
+
+To configure this module, you need to:
+
+	No configuration needed
+
+
+
```
Comment (houssine78) [inline account_invoice_journal_filter/README.rst:27]: maybe obvious instruction and empty section could be removed

**Pull request closed without merging**